components/filter/filter-dimension-set.js

```diff
@@ -18,6 +18,15 @@ class FilterDimensionSet extends LitElement {
 			 * Whether the values for this dimension are still loading and a loading spinner should be displayed
 			 */
 			loading: { type: Boolean },
+			/**
+			 * Whether to hide the search input, fire an event on search, or perform a simple text search
+			 * @type {'none'|'event'|'simple'}
+			 */
+			searchType: { type: String, attribute: 'search-type' },
```

Comment:
Thoughts about `none`, `manual` and `automatic`?

Reply:
Yeah I like that, I think I could then cleanup that mapping function too to be simpler